Spanner/tests/Unit/DatabaseTest.php

```diff
@@ -333,6 +333,27 @@ public function testCreate()
         $this->assertInstanceOf(LongRunningOperation::class, $op);
     }
 
+    /**
+     * @group spanner-admin
+     */
+    public function testUpdateDatabase()
```

Comment:
testUpdateDatabase() -> testUpdate()

Reply:
we have another function called `testUpdate` in Database class which updates the schema of a table. Hence cannot use that name.